report/assets/styles.css

```diff
@@ -482,9 +482,7 @@
   border-radius: 3px;
   color: var(--link-color);
   background-color: var(--report-background-color);
-  padding: var(--default-padding);
-  font-size: var(--report-font-size-secondary);
-  line-height: var(--report-line-height-secondary);
+  margin: 5px;
```

Comment:
fixes this
![image](https://user-images.githubusercontent.com/4071474/141389825-3d969459-e8b7-4b21-8fec-eb94bc468950.png)


was changed here  https://github.com/GoogleChrome/lighthouse/pull/13249/files#diff-32ea75d44d2425272f7c1c7ceb52d47e5c4bb78cb8facbb55495a4d0d7658729R479

`margin` was important here. Put to 5 instead of the original 10.

fontsize and lineheight seemed to have to effect so I removed.

Reply:
now
![image](https://user-images.githubusercontent.com/4071474/141389921-7f2708f7-3286-41aa-9b64-77b72a099373.png)